pinot-query-planner/src/test/resources/queries/LiteralEvaluationPlans.json

```diff
@@ -259,5 +259,49 @@
         "expectedException": ".*No match found for function signature nonExistFun.*"
       }
     ]
+  },
+  "literal_planning_cte_test": {
+    "comment": "Tests for CTEs in literal planning. The SQL parser cannot get rid of expressions that cross CTE, so this is useful to check that the expressions are simplified in the logical plan.",
+    "queries": [
+      {
+        "description": "Simple filter on constants is simplified",
+        "sql": "EXPLAIN PLAN FOR WITH CTE_B AS (SELECT 1692057600000 AS __ts FROM a GROUP BY __ts) SELECT 1692057600000 AS __ts FROM CTE_B WHERE __ts >= 1692057600000 GROUP BY __ts",
+        "output": [
+          "Execution Plan",
+          "\nLogicalProject(__ts=[1692057600000:BIGINT])",
+          "\n  LogicalAggregate(group=[{0}])",
+          "\n    PinotLogicalExchange(distribution=[hash[0]])",
+          "\n      LogicalAggregate(group=[{0}])",
+          "\n        LogicalProject(__ts=[1692057600000:BIGINT])",
+          "\n          LogicalTableScan(table=[[default, a]])",
+          "\n"
```

Comment:
This query works in master. Our `PinotEvaluateLiteralRule.Filter.INSTANCE` is able to get rid of it.

Reply:
This should no longer be an issue (https://github.com/apache/pinot/pull/13711).